installers/rpm-code/codes/flash.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+flash_main() {
+    codes_dependencies hypre
+    # TODO(e-carlin):  is $install_server right? rn probably already has an abstraction for this
+    # TODO(e-carlin):  need to make this url secret with the secret coming from the env
+    codes_download "$install_server/flash-4.6.2.tar.gz" "FLASH4.6.2"
```

Comment:
@robnagler is there an abstraction already for this? I couldn't find one... should one be created?

How should we share the secret part of the url? Also should the code live at $install_server/flash? Do we need a dev/alpha/beta part of the path?

Reply:
With #97 this will be: 

```py
codes_download_proprietary "$install_server/flash-4.6.2.tar.gz" "FLASH4.6.2"
```

`install_proprietary_key` is the magic value, and it is cascaded by `install_vars_export` for `vagrant ssh` and `docker run` used by build scripts.

Reply:
For the build, we do not need an alpha/beta/etc. That's promotion, and handled by our distro mechanisms and rsconf.